acceptance/ci-inputs/kind-inputs.yaml

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) HashiCorp, Inc.
 # SPDX-License-Identifier: MPL-2.0
 
-kindVersion: v0.22.0
+kindVersion: v0.23.0
 kindNodeImage: kindest/node:v1.29.2@sha256:51a1434a5397193442f0be2a297b488b6c919ce8a3931be0ce822606ea5ca245
```

Comment:
This will need to be bumped as well as it's what actually controls the K8s version - we can use the latest:

`kindest/node:1.30.2@sha256:ecfe5841b9bee4fe9690f49c118c33629fa345e3350a0c67a5a34482a99d6bba`